### ISSUE TITLE
Flip question order

### ISSUE THREAD

**Issue by h4n5h4n5** [rstammer/kneipenquiz#9]
Nummerierung der Fragen so verändern, dass neue Fragen oben stehen und für anders Eingeloggte, diese gekennzeichnet sind (durch fettgedruckte Nummer z.B.).
Werden bereits gelöschte Fragen mitgezählt?

**Comment by rstammer** [comment]
Ja, die werden mitgezählt! Die Nummerierung an sich zu verändern ist nicht möglich (bzw wäre ein sehr unvorteilhafte Eingriff), weil das die ID's in der Datenbank sind.

Besser wäre: Ich füge eine Filtermaske hinzu um z.B. nach Tags, Erstelldatum oder Kategorie zu filtern. Dementsprechend würden dann nur Fragen angezeigt werden, die der Filter-Eingabe entsprechen - Das 'Sucherebnis' kann man natürlich auf dynamische Weise sortieren (z.B. nach Erstelldatum). Das wäre im Sinne des Erfinders, oder?

**Comment by h4n5h4n5** [comment]
Ja, wäre es.
